dev/docker-compose.yml

```diff
@@ -113,8 +113,8 @@ services:
 
   oidc-phantomjs:
     environment:
-      - KEYCLOAK_USER=alice
-      - KEYCLOAK_PASSWORD=alice
+      - KEYCLOAK_APP_USER=alice
+      - KEYCLOAK_APP_USER_PASSWORD=alice
```

Comment:
does this work? I think that `KEYCLOAK_USER` & `KEYCLOAK_PASSWORD` are required for setting the adminsitrator user in keycloak.

Please verify that the tests pass in the `./cli exec` flow and not only `ci/test`

Reply:
I test them both and they are working please double check me if you are thinking that i am missing something

Reply:
sorry i thought these were params of the `oidc-keycloak` container